nuitka/tools/release/Documentation.py

```diff
@@ -167,23 +160,20 @@ def makeManpage(python, suffix):
     makeManpage("python3", "3")
 
 
+@which("rst2pdf")
 def createRstPDF(document, args):
-    assert (
-        subprocess.call(
-            "%(rst2pdf)s %(args)s  %(document)s"
-            % {
-                "rst2pdf": (
-                    "rst2pdf"
-                    if os.name != "nt"
-                    else r"C:\Python27_32\Scripts\rst2pdf.exe"
-                ),
-                "args": " ".join(args),
-                "document": document,
-            },
-            shell=True,
-        )
-        == 0
-    ), document
+    subprocess.check_call(
```

Comment:
I guess, I was lazy here. But obviously, I should simply demand that rst2pdf be in the path when running the tool, and not make up of it like that.

Reply:
Yes, that can be simplified. I changed this call too.